specification/networkanalytics/resource-manager/Microsoft.NetworkAnalytics/stable/2023-11-15/networkanalytics.json

```diff
@@ -0,0 +1,2571 @@
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataProductsCatalogs_ListByResourceGroup_MaximumSet_Gen": {
+            "$ref": "./examples/DataProductsCatalogs_ListByResourceGroup_MaximumSet_Gen.json"
+          },
+          "DataProductsCatalogs_ListByResourceGroup_MinimumSet_Gen": {
+            "$ref": "./examples/DataProductsCatalogs_ListByResourceGroup_MinimumSet_Gen.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetworkAnalytics/dataProductsCatalogs/default": {
+      "get": {
+        "operationId": "DataProductsCatalogs_Get",
+        "tags": [
+          "DataProductsCatalogs"
+        ],
+        "description": "Retrieve data type resource.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/DataProductsCatalog"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataProductsCatalogs_Get_MaximumSet_Gen": {
+            "$ref": "./examples/DataProductsCatalogs_Get_MaximumSet_Gen.json"
+          },
+          "DataProductsCatalogs_Get_MaximumSet_Gen - generated by [MinimumSet] rule_MinimumSet_Gen": {
+            "$ref": "./examples/DataProductsCatalogs_Get_MinimumSet_Gen.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AccountSas": {
+      "type": "object",
+      "description": "The details for storage account sas creation.",
+      "properties": {
+        "startTimeStamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Sas token start timestamp."
+        },
+        "expiryTimeStamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Sas token expiry timestamp."
+        },
+        "ipAddress": {
+          "type": "string",
+          "description": "Ip Address"
```

Comment:
>Address [](http://example.com/codeflow?start=29&length=7)

Since this is a required property, I think documenting it + its importance more fully could be a good idea!

Reply:
This is cloned from private repo. We will first make required change in the private repo and make a fix in public repo in the next release.